Gemfile

```diff
@@ -37,7 +37,11 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 gem 'uglifier'
 
 group :development, :test do
+  gem 'capybara'
+  gem 'database_cleaner-active_record'
+  gem 'factory_bot_rails'
   gem 'rspec-rails'
+  gem 'selenium-webdriver'
 end
```

Comment:
Maybe in test group only? `rspec-rails` should be in both dev & test as per their documentation. What about the new dependencies?